client/client_interfaces.go

```diff
@@ -116,20 +116,32 @@ type ImageAPIClient interface {
 	ImageHistory(ctx context.Context, image string, opts image.HistoryOptions) ([]image.HistoryResponseItem, error)
 	ImageImport(ctx context.Context, source image.ImportSource, ref string, options image.ImportOptions) (io.ReadCloser, error)
 
-	// ImageInspectWithRaw returns the image information and its raw representation.
-	//
-	// Deprecated: Use [Client.ImageInspect] instead. Raw response can be obtained using the [ImageInspectWithRawResponse] option.
-	ImageInspectWithRaw(ctx context.Context, image string) (image.InspectResponse, []byte, error)
 	ImageInspect(ctx context.Context, image string, _ ...ImageInspectOption) (image.InspectResponse, error)
 	ImageList(ctx context.Context, options image.ListOptions) ([]image.Summary, error)
-	ImageLoad(ctx context.Context, input io.Reader, opts image.LoadOptions) (image.LoadResponse, error)
+	ImageLoadWithOptions(ctx context.Context, input io.Reader, opts image.LoadOptions) (image.LoadResponse, error)
```

Comment:
Could we not still have `ImageLoad` but use variadic opts like ImageInspect?

Reply:
We can, but it won't fix `bool` vs `image.LoadOptions()` (if those would be functional arguments) signature